src/libraries/System.Security.Cryptography.Algorithms/src/Resources/Strings.resx

```diff
@@ -363,4 +363,4 @@
   <data name="Cryptography_FeedbackSizeNotSupported" xml:space="preserve">
     <value>The current platform does not support the specified feedback size.</value>
   </data>
-</root>
+</root>
```

Comment:
Please add the newline back to this file so it doesn't get an unnecessary edit applied.